src/common/components/Input.tsx

```diff
@@ -60,9 +61,10 @@ export const Input = forwardRef<HTMLInputElement, InputInterface>(
           {...rest}
           className={classes.input}
           id={idForLabel}
-          type={'text'}
-          onFocus={handleFocus}
+          maxLength={maxLength}
           onBlur={handleBlur}
+          onFocus={handleFocus}
+          type={'text'}
```

Comment:
If these should be "defaults" and overridable by props on the Input element, they should be above the {...rest} call, so that they get overridden

Reply:
IIRC

Reply:
Yes, I think you're right and at least `maxLength` should go above `{...rest}` I'm not sure about the others, it depends on how much of our will we want to impose on users of this component "by default" since if one needed an override they could fix it here too.